src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+- [Discover Services & Characteristics](https://googlechrome.github.io/samples/web-bluetooth/discover-services-and-characteristics.html) - discover all accessible primary services and their characteristics from a BLE Device.
+- [Automatic Reconnect](https://googlechrome.github.io/samples/web-bluetooth/automatic-reconnect.html) - reconnect to a disconnected BLE device using an exponential backoff algorithm.
+- [Read Characteristic Value Changed](https://googlechrome.github.io/samples/web-bluetooth/read-characteristic-value-changed.html) - read battery level and be notified of changes from a BLE Device.
+- [Read Descriptors](https://googlechrome.github.io/samples/web-bluetooth/read-descriptors.html) - read all characteristic's descriptors of a service from a BLE Device.
+- [Write Descriptor](https://googlechrome.github.io/samples/web-bluetooth/write-descriptor.html) - write to the descriptor "Characteristic User Description" on a BLE Device.
+
+Check out our [curated Web Bluetooth Demos] and [official Web Bluetooth Codelabs] as well.
+
+## Libraries
+
+- [web-bluetooth-utils] is a npm module that adds some convenience functions to
+  the API.
+- A Web Bluetooth API shim is available in [noble], the most popular Node.js BLE
+  central module. This allows you to webpack/browserify noble without the need
+  for a WebSocket server or other plugins.
+- [angular-web-bluetooth] is a module for [Angular] that abstracts away all the
+  boilerplate needed to configure the Web Bluetooth API.
+- [&lt;platinum-bluetooth>] is a set of [Polymer] elements to discover and
+  communicate with nearby Bluetooth devices based on the Web Bluetooth API. For
+  instance, here's how to read battery level from a nearby bluetooth device
+  advertising a Battery service:
+  
+```js
+<platinum-bluetooth-device services-filter='["battery_service"]'>
+  <platinum-bluetooth-service service='battery_service'>
+    <platinum-bluetooth-characteristic characteristic='battery_level'>
+    </platinum-bluetooth-characteristic>
+  </platinum-bluetooth-service>
+</platinum-bluetooth-device>
+```
+
+```js
+const bluetoothDevice = document.querySelector('platinum-bluetooth-device');
+const batteryLevel = document.querySelector('platinum-bluetooth-characteristic');
+
+bluetoothDevice.request()
+.then(_ => batteryLevel.read())
+.then(value => {
+  console.log('Battery percentage is ' + value.getUint8(0));
+})
+.catch(error => { console.log(error); });
+```
+
+## Tools
+
+- [Get Started with Web Bluetooth] is a simple Web App that will generate all
+  the JavaScript boilerplate code to start interacting with a Bluetooth device.
+  Enter a device name, a service, a characteristic, define its properties and
+  you're good to go.
+- If you're already a Bluetooth developer, the [Web Bluetooth Developer Studio
+  Plugin] will also generate the Web Bluetooth JavaScript code for your
+  Bluetooth device.
+
+## Dev Tips
```

Comment:
```suggestion
## Tips
```
Considering that our site is targeted at web developers I don't think we need to preface it as `Dev Tips`

Reply:
Done